tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -405,6 +405,8 @@ def export_tag_to_basic_filenames_and_read_urls(
     def export_tag_to_basic_filenames(
         self, dataset_id: str, tag_id: str, **kwargs
     ) -> str:
+        if kwargs["page_offset"] and kwargs["page_offset"] > 0:
```

Comment:
Please explain what this is for?

Reply:
Probably because in the pagination case, something else has to be returned, so that the test does not break.